.github/workflows/clang-tidy.yaml

```diff
@@ -0,0 +1,66 @@
+name: clang-tidy
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    # only at 5AM Saturday
+    - cron: '0 5 * * SAT'
+  workflow_dispatch:
+    inputs:
+      build_type:
+        description: 'CMake build type'
+        required: false
+        default: 'RelWithDebInfo'
+        type: choice
+        options:
+          - Debug
+          - RelWithDebInfo
+          - Dev
+          - Sanitize
+          - Coverage
+      clang_tidy_checks:
+        description: 'clang-tidy --checks options'
+        required: false
+        default: 'bugprone-use-after-move'
+        type: string
```

Comment:
I was running --checks=-*,bugprone-use-after-move to remove noise from default options, so we're really focusing on use-after-move only, and eventually we can incrementally enable others that we find useful.

Reply:
thanks. sure. added `-*`.